src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -944,7 +947,31 @@ export class HaAutomationEditor extends PreventUnsavedMixin(
 
         // wait for automation to appear in entity registry when creating a new automation
         if (entityRegPromise) {
+          const timeout = setTimeout(() => {
+            this._readOnly = true;
+            this._dirty = false;
+            showAlertDialog(this, {
+              title: this.hass.localize(
+                "ui.panel.config.automation.editor.new_automation_setup_failed_title",
+                {
+                  type: this.hass.localize(
+                    "ui.panel.config.automation.editor.type_automation"
+                  ),
+                }
+              ),
+              text: this.hass.localize(
+                "ui.panel.config.automation.editor.new_automation_setup_failed_text",
+                {
+                  type: this.hass.localize(
+                    "ui.panel.config.automation.editor.type_automation"
+                  ),
+                }
+              ),
+              warning: true,
+            });
+          }, 5000);
```

Comment:
5 seconds seems long, I think we make this fail quicker?

Reply:
Have a favorite number? I would like to be careful not to fire prematurely for someone just on a slow connection, but I don't know what is standard for such things.

Reply:
Yeah, I'm mainly worried because we don't show any progress indicator or what so ever, so if I was looking at this screen for 5 seconds I would be whats happening?! In general the rule of thumb is that if it takes longer than a second we should show a progress indicator.

I would normally say this action _should_ be done within a second, but let's set it at 2 seconds?